lib/components/form-fieldset.vue

```diff
@@ -27,12 +38,15 @@
                 return this.horizontal ? ('col-sm-' + (12 - this.labelSize)) : 'col-12';
             }
         },
-        mounted() {
-            const content = this.$refs.content;
-            if (!content) {
-                return;
+        methods: {
+            target() {
```

Comment:
I could also alter the default `inputSelector` value to include `.dropdown` and the check for its ID existance. And in dropdown, we could generate a default `b_UID` if an ID is not given.